daemon/inertiad/project/data_test.go

```diff
@@ -65,6 +65,49 @@ func TestDataManager_EnvVariableOperations(t *testing.T) {
 	}
 }
 
+func TestDataManager_ProjectBuildDataOperations(t *testing.T) {
+	type args struct {
+		projectName string
+		metadata    DeploymentMetadata
+		numProjects int
+	}
+	tests := []struct {
+		name    string
+		args    args
+		wantErr bool
+	}{
+		{"valid project build", args{"projectB", DeploymentMetadata{"hash", "ID", "status", "time"}, 2}, false},
```

Comment:
might want to add some potential error cases here - what if the project name is empty? etc.